### PR TITLE
Yuan/add doc for k8s config

### PR DESCRIPTION
**Pull request by hao1939**
Hi @ydye @hwuu ,

Please helping review my doc update.
This PR include:
1. Explanation about kubernetes-configuration.yaml/docker-registry.
2. Update about how to use prebuild 'dev-box' image.

### CODE REVIEW

File: pai-management/doc/how-to-setup-dev-box.md [hunk 1]
```diff
@@ -1,11 +1,40 @@
...
 
-#### Build the Dev-Box's docker image.
+## Use prebuild dev-box image
+
+```bash
+
+# Pull the dev-box image from Docker Hub
+sudo docker pull docker.io/openpai/dev-box
+
+# Run your dev-box
```
Comment (fanyangCS) [inline pai-management/doc/how-to-setup-dev-box.md]: please add a comment:

1. Assume the path of ```custom-hadoop-binary-path``` in your service-configuration is ```/pathHadoop```
2. Assume the directory path of your cluster-configuration is ``/pathConfiguration`

Comment (hao1939) [reply pai-management/doc/how-to-setup-dev-box.md]: Thanks, I will fix it with a new commit.

File: pai-management/doc/how-to-setup-dev-box.md [hunk 2]
```diff
@@ -1,11 +1,43 @@
...
+# Assume the path of custom-hadoop-binary-path in your service-configuration is /pathHadoop.
+# Assume the directory path of your cluster-configuration is /pathConfiguration.
+# By the way, both of the above could be an empty directory.
```
Comment (fanyangCS) [inline pai-management/doc/how-to-setup-dev-box.md]: >By the way, both of the above could be an empty directory. [](start = 2, length = 58)

if you would like to say so. you better explain what empty directory means (what the behavior? if they are empty).

Comment (fanyangCS) [reply pai-management/doc/how-to-setup-dev-box.md]: I suggest we do not say so. do not give users the freedom they are not sure how to leverage.

---
In reply to: [196300305](https://github.com/Microsoft/pai/pull/739#discussion_r196300305) [](ancestors = 196300305)

Comment (hao1939) [reply pai-management/doc/how-to-setup-dev-box.md]: Yes, you are right. My express is misleading.
Actually I want to tell user that they don't need to modify it.

**Pull request merged**